certify-service/src/test/resources/application-test.properties

```diff
@@ -57,12 +57,12 @@ mosip.certify.key-values={\
               'credential_issuer': '${mosipbox.public.url}', 	\
               'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/issuance/credential', \
               'display': {{'name': 'e-Signet', 'locale': 'en'}},\
-              'credentials_supported' : { \
+              'credential_configurations_supported' : { \
                  "SampleVerifiableCredential_ldp" : {\
                     'format': 'ldp_vc',\
                     'scope' : 'sample_vc_ldp',\
                     'cryptographic_binding_methods_supported': {'did:jwk'},\
-                    'cryptographic_suites_supported': {'RsaSignature2018'},\
+                    'credential_signing_alg_values_supported': {'RsaSignature2018'},\
                     'proof_types_supported': {'jwt'},\
```

Comment:
proof_types_supported this change is missing and we should have test case or assert statements.

Reply:
Same is missed in the docker compose also